.eslintrc.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
     "plugin:react/recommended",
     "plugin:flowtype/recommended",
   ],
-  parser: "babel-eslint",
+  parser: "@babel/eslint-parser",
```

Comment:
😒 that was it? all that syntax and ubuntu search and it was a dash? 

Lord, I owe you 🍺  @ofqwx

Reply:
@twblack88 no, it wasn't just a dash. `babel-eslint` is now `@babel/eslint-parser` (https://github.com/babel/babel-eslint), and yes this new version solves the issue of eslint not able to find the plugin on some platforms. but migrating to this new version is not only the dash, it requires to have a babel config file and since this is a gatsby project, the babel configuration is handled internally, so I had to do a babel config file that includes gatsby presets, eslint presets and flow types presets, that's why now we have `.babelrc` in the root of the project.